Jenkinsfile

```diff
@@ -71,7 +80,7 @@ pipeline {
                     }
                     steps {
                         echo 'Beginning tests...'
-                        bat 'C:\\Users\\root\\.cargo\\bin\\cargo test --all'
+                        bat 'C:\\Users\\root\\.cargo\\bin\\cargo test --all --features "vulkan sdl_controller json saveload shader-compiler"'
```

Comment:
why is `shader-compiler` enabled here?

Reply:
Because I like to copy and paste things :P